go-controller/pkg/kubevirt/types.go

```diff
@@ -0,0 +1,5 @@
+package kubevirt
+
+const (
+	AllowPodBridgeNetworkLiveMigrationAnnotation = "kubevirt.io/allow-pod-bridge-network-live-migration"
```

Comment:
Good catch.

Reply:
The change proposed at the enhancement to skip ip config at CNI has being implemented at different commit because I have change it a little instead of a  global annotation have use the pod network status so we can skip ip per network:
- https://github.com/ovn-org/ovn-kubernetes/pull/3478/commits/77da75ebefa99fc3cc9eb83cc609a9b857b7c1b0

Reply:
then I don't understand the purpose of this commit; its goal (from the commit msg) is:
> The hypershift worker will be mark with the kubevirt label to state that
default pod network is live migratable and the address will be set using
DHCP options. This change skip ip config for this case.

Reply:
yeah lets rework the commits so they make sense and just arent added on top of each other based on the history of the enhancement changes. Let's make it generic (not tied to kubevirt or VMs) since it is ovn.org/skip-ip-configuration-on-cni.

Reply:
> yeah lets rework the commits so they make sense and just arent added on top of each other based on the history of the enhancement changes. Let's make it generic (not tied to kubevirt or VMs) since it is ovn.org/skip-ip-configuration-on-cni.

Squashed and remove kubevirt references from this skip ip config commit.

Reply:
> What do you mean w/ activate DHCP ? Install the ovn logical flows that assign IPs to the workloads via DHCP ?
> 
> If so, yes.

After talks offline Miguel this will not configure the DHCP flows

Reply:
Right; for clarity: the DHCP flows will be installed based on the network's subnet configuration; this means the default network will always install it (not exactly sure from where) - but the secondary networks will depend on the `subnets` configuration.